rc/extra/elixir.kak

```diff
@@ -0,0 +1,85 @@
+# http://elixir-lang.org
+# ----------------------
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.](ex|exs) %{
+    set buffer filetype elixir
+}
+
+# Faces
+# -----
+
+face ElixirAtom yellow
+face ElixirUnderscore white+d
+face ElixirAttribute green+d
+face ElixirSpecialKeyword blue+b
+face ElixirEnv red+b
+face ElixirOperator rgb:a8a8a8
+face ElixirInterpolation rgb:00d7ff
```

Comment:
It felt a bit dirty to add them indeed, but I wanted to have a bit more colors to select from, but the fact different colorschemes are now effectively not supported for the Elixir highlighter is indeed not good.

I'll commit a version without the extra faces.